dask/dataframe/tests/test_dataframe.py

```diff
@@ -1704,6 +1683,9 @@ def test_combine_first():
     assert_eq(ddf1.B.combine_first(df2.B), df1.B.combine_first(df2.B))
 
 
+@pytest.mark.filterwarnings(
+    "ignore:The 'freq' argument in Timestamp is deprecated:FutureWarning"
+)  #  PANDAS_GT_130 https://github.com/pandas-dev/pandas/issues/41949
```

Comment:
This seems fine as this ignore is quite specific. We could do some version checking to only ignore for certain versions of `pandas`, but that may be overkill.

As a side note, if https://github.com/pandas-dev/pandas/issues/41949 gets in prior to a release, then we can come back and remove any ignoring here.